.changeset/twenty-colts-invite.md

```diff
@@ -0,0 +1,6 @@
+---
+'@compiled/parcel-transformer': minor
```

Comment:
Do you think if this change should be `minor`?

Reply:
yeah probably a good idea since we're potentially changing resolving behaviour by default